src/client/app/warehouses/warehouses.controller.js

```diff
@@ -1,22 +1,30 @@
-(function() {
+(function () {
   'use strict';
 
   angular
     .module('app.warehouses')
     .controller('WarehousesController', WarehousesController);
 
-  WarehousesController.$inject = ['warehouses', 'logger'];
+  WarehousesController.$inject = ['warehouses', 'logger', '$location', '$state'];
   /* @ngInject */
-  function WarehousesController(warehouses, logger) {
+  function WarehousesController(warehouses, logger, $location, $state) {
     var vm = this;
 
     vm.title = 'Warehouses';
+    vm.supplies = [];
+    vm.location = $location;
+
 
     activate();
 
     function activate() {
-      warehouses.getWarehouses().then(function(data) {
+      warehouses.getWarehouses().success(function (data) {
```

Comment:
oh, success is deprecated. use `then`

`The $http legacy promise methods success and error have been deprecated. Use the standard then method instead. If $httpProvider.useLegacyPromiseExtensions is set to false then these methods will throw $http/legacy error.`